plugins.xml

```diff
@@ -12669,4 +12669,108 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="pt_BR">Garante que a pergunta esteja relacionada ao revista atual.</description>
 		</release>
 	</plugin>
+	<plugin category="oaiMetadataFormats" product="biblat">
+		<name locale="es">Formato BIBLAT de Metadatos</nam
+		<name locale="en">BIBLAT Metadata Format</name>
+		<homepage>https://biblat.unam.mx/en/</homepage>
+
+		<summary locale="es">Estructura los metadatos de forma que son consistentes con el formato BIBLAT.</summary>
+		<summary locale="en">Provides a metadata structure consistent with the BIBLAT metadata format.</summary>
+
+		<description locale="es"><![CDATA[<p>Este modulo aggrega un formato de los metadatos para Bibliografía Latinoamericana.</p>]]></description>
+		<description locale="en"><![CDATA[<p>This module adds a metadata format for Bibliografía Latinoamericana.</p>]]></description>
+
+		<maintainer>
+			<name>Bibliografía Latinoamericana</name>
+			<institution>UNAM</institution>
+			<email>biblat@dgb.unam.mx</email>
+		</maintainer>
+		
+		<release date="2024-01-01" version="3.0" md5="3234a0dd82158ce104dfcf91f070e9a6">
+			<package>https://github.com/dgb-sistemas/plugin_biblat/releases/download/v3.0/biblat.tar.gz</package>
+			<compatibility application="ojs3">
```

Comment:
Aquí, y en cada `compatibility`, `application` debe ser "ojs2".  La etiqueta sigue "ojs2" mientras la versión es 3.x.  
Here, and in each `compatibility`, `application` should be "ojs2".  The tag remains "ojs" while the version is 3.x

Reply:
Tú código de ejemplo tenía ojs2 y pensé que al tratarse de la versión 3 debía cambiar, pero ya lo corregí como indicas